src/app/world-map/world-map.component.ts

```diff
@@ -0,0 +1,34 @@
+import { AfterViewInit, Component } from '@angular/core';
+import * as L from 'leaflet';
+
+@Component({
+  selector: 'app-world-map',
+  templateUrl: './world-map.component.html',
+  styleUrls: ['./world-map.component.scss'],
+})
+export class WorldMapComponent implements AfterViewInit {
+  private map;
+
+  constructor() {}
+
+  ngAfterViewInit(): void {
+    this.initMap();
+  }
+
+  private initMap(): void {
+    this.map = L.map('map', {
+      center: [51.0, 0.0],
+      zoom: 3,
+    });
+    const tiles = L.tileLayer(
+      'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png',
```

Comment:
This url should be pulled out too. What if we wanted to use a different map provider?